tests/conftest.py

```diff
@@ -65,6 +39,44 @@ def data_unmerged():
     return load_dataset(datapath)
 
 
+@pytest.fixture
+def hkls(data_merged):
+    """
+    Return all Miller indices with H, K, L values between [-5, 5]
+    """
+    return data_merged.hkls
+
+
+@pytest.fixture
+def dataset_hkl(data_merged):
+    """
+    Build DataSet for testing containing only Miller indices
+    """
+    H = data_merged.hkls
+    dataset = rs.DataSet({"H": H[:, 0], "K": H[:, 1], "L": H[:, 2]})
+    dataset.set_index(["H", "K", "L"], inplace=True)
+    return dataset
```

Comment:
Is `data_merged` 100% complete between [-5, 5] for all hkls? I found this to be a useful fixture for testing things like gemmi/sg operations in the past, so I hesitate to fully change it over to this, given that `data_merged` is an experimental HEWL dataset and not a complete synthetic one.

Sorry to be a pain, but I think this was better as it was before -- I worry a bit about collateral effects of changing these fixtures in this way.

Reply:
okay, it makes sense to be careful here. i reverted this change. i simplified the test code a little. that seems like a better solution than dataset-specific hkl fixtures.